<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gradientForm</class>
 <widget class="QDialog" name="gradientForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>330</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Gradients</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_1">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="color1Button">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="pos1SpinBox">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="color2Button">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="pos2SpinBox">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="color3Button">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="pos3SpinBox">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="color4Button">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="pos4SpinBox">
         <property name="maximumSize">
          <size>
           <width>45</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="dialLabel">
         <property name="text">
          <string>Direction</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dial">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
         <property name="sliderPosition">
          <number>90</number>
         </property>
         <property name="invertedAppearance">
          <bool>true</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchTarget">
          <double>10.000000000000000</double>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="gradientGroupBox">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Gradient type</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="noGradientButton">
          <property name="text">
           <string>None</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="linearGradientButton">
          <property name="text">
           <string>Linear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radialGradientButton">
          <property name="text">
           <string>Radial</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="conicalGradientButton">
          <property name="text">
           <string>Conical</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="previewLabel">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>100</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>255</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>255</green>
             <blue>127</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>255</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>gradientForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>gradientForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
